src/layout/mod.rs

```diff
@@ -1320,6 +1320,41 @@ impl<W: LayoutElement> Layout<W> {
         monitor.focus_window_or_workspace_up();
     }
 
+    pub fn maybe_focus_window_or_monitor_left(&mut self, output: &Output) -> bool {
+        let Some(monitor) = self.active_monitor() else {
+            return false;
```

Comment:
I guess what we care about is whether we focused an output or not. So this should `return true;`. Alternatively, I suggest flipping the return value, so this will return `false`, focus_left will return `false` and only focus_output will return `true`.